doc/source/visualisation.rst

```diff
@@ -1,4 +1,145 @@
+.. include:: include/global.rst
+
+=======================
 Visualisation of graphs
 =======================
+|igraph| includes functionality to visualize graphs. There are two main components: graph layouts and graph plotting.
+
+In the following examples, we will assume |igraph| is imported as `ig` and a
+:class:`Graph` object has been previously created, e.g.:
+
+>>> import igraph as ig
+>>> g = ig.Graph(edges=[[0, 1], [2, 3]])
+
+Read the `API documentation`_ for details on each function and class. The `tutorial`_ contains examples to get started.
+
+Graph layouts
+=============
+A graph *layout* is a low-dimensional (usually: 2 dimensional) representation of a graph. Different layouts for the same
+graph can be computed and typically preserve or highlight distinct properties of the graph itself. Some layouts only make
+sense for specific kinds of graphs, such as trees.
+
+|igraph| offers several graph layouts. The general function to compute a graph layout is :meth:`Graph.layout`:
+
+>>> layout = g.layout(layout='auto')
+
+See below for a list of supported layouts. The resulting object is an instance of `igraph.layout.Layout` and has some
+useful properties:
+
+- :attr:`Layout.coords`: the coordinates of the vertices in the layout (each row is a vertex)
+- :attr:`Layout.dim`: the number of dimensions of the embedding (usually 2)
+
+and methods:
+
+- :meth:`Layout.boundaries` the rectangle with the extreme coordinates of the layout
+- :meth:`Layout.bounding_box` the boundary, but as an `igraph.drawing.utils.BoundingBox` (see below)
+- :meth:`Layout.centroid` the coordinates of the centroid of the graph layout
+
+Indexing and slicing can be performed and returns the coordinates of the requested vertices:
+
+>>> coords_subgraph = layout[:2]  # Coordinate of the first vertex
```

Comment:
`layout[:2]` should return the coordinates of the first two vertices; `layout[0]` is the one that returns the coordinates of the first vertex.

Reply:
fixed